File: 백준/Bronze/31867. 홀짝홀짝/홀짝홀짝.c
#include <stdio.h>

int main(void) {
  int a;
  scanf("%d", &a);

  char str[a + 1];
  scanf("%s", str);
  int even = 0, odd = 0;

  for (int i = 0; i < a; i++){
    if((str[i] - '0') % 2 == 0){
      even++;
    }
    else{
      odd++;
    }
  }

  if(even > odd){
    printf("0");
  }
  else if(even < odd){
    printf("1");
  }
  else{
    printf("-1");
  }
  return 0;
}

File: 백준/Bronze/31867. 홀짝홀짝/홀짝홀짝.py
a = int(input())
b = input()

even, odd = 0, 0
for item in b:
    if int(item) % 2 == 0:
        even += 1
    else:
        odd += 1

if even > odd:
    print(0)
elif odd > even:
    print(1)
else:
    print(-1)